ios-xcframework/build.sh

```diff
@@ -77,25 +100,21 @@ PLATFORM_SIMULATOR=iphonesimulator
 archive $PLATFORM_IOS
 archive $PLATFORM_SIMULATOR
 
-# 2. Create XCFrameworks for every framework in the archives
+# 2. Flatten embedded frameworks for every framework in the archives
+flattenFrameworks $PLATFORM_IOS
+flattenFrameworks $PLATFORM_SIMULATOR
+
+# 3. Create XCFrameworks for every framework in the archives
 #
 # Notice how we loop on $PLATFORM_IOS as a way to get the frameworks for both platforms.
 # We could use either platform to achieve the same result.
-for FRAMEWORK in $(find "$ARCHIVES_ROOT/$PLATFORM_IOS.xcarchive/Products/Library/Frameworks" -type d -name "*.framework");
+for FRAMEWORK in "$ARCHIVES_ROOT/$PLATFORM_IOS.xcarchive/Products/Library/Frameworks"/*;
```

Comment:
This change implies that we'll only loop over the frameworks at the root level. Embedded frameworks (i.e. nested in framework folders) will be skipped as it should have been processed in the previous step.